### DISCUSSION TITLE
How can I add a Discord widget (image) to the footer

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2181] [Q&A]
I would like to add the widget image from Discord (https://discord.com/api/guilds/:id/widget.png?style=banner2) into the footer of the docs, but I'm not sure if and where I would put it.

I already use theme extension to change the footer slightly (changing the info text to be a bit more detailed. See image), so I could add it there.
My idea for styling was to move the social icons from the right to the centre and add the discord widget to the right, but again do I have no idea how to achieve this in the first place.

Here is an image of the footer as it is right now:
![image](https://user-images.githubusercontent.com/11576465/103640363-244df980-4f50-11eb-864a-0b32d10fe706.png)

And here is the current main.html I use to override stuff:
https://github.com/purrbot-site/Docs/blob/master/theme/main.html

If you have any other suggestions including those on how to improve the setup of the theme-extension, let me know.

**Comment by facelessuser** [comment]
Can't you just use the built-in social support? https://github.com/facelessuser/pymdown-extensions/blob/master/mkdocs.yml#L141

**Reply by Andre601** [reply]
That's the icon and not what I want.

**Reply by facelessuser** [reply]
For reference, it is documented here: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#social-links

**Reply by facelessuser** [reply]
It is not clear what you want then. I followed the link to the "widget image", but just got `{"guild_id": ["Value \":id\" is not snowflake."]}`. I'm not sure what that is.

**Reply by Andre601** [reply]
Replace it with the ID of a guild (Which has the widget setting enabled)

for example becomes https://discord.com/api/guilds/81384788765712384/widget.png?style=banner2 this:
![widget](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner2)

**Reply by facelessuser** [reply]
Okay, sounds like you are going to need some custom override of the footer. Unfortunately, I don't have time to look into that, but maybe someone else can answer.
